samples/model-builder/create_training_pipeline_image_classification_sample.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from google.cloud import aiplatform
+
+#  [START aiplatform_sdk_create_training_pipeline_image_classification_sample]
+def create_training_pipeline_image_classification_sample(
+    project: str,
+    display_name: str,
+    dataset_id: int,
+    model_display_name: str = None,
+    training_fraction_split: float = 0.8,
+    validation_fraction_split: float = 0.1,
+    test_fraction_split: float = 0.1,
+    budget_milli_node_hours: int = 8000,
+    disable_early_stopping: bool = False,
+    sync: bool = True,
+):
+    aiplatform.init(project=project)
+
+    job = aiplatform.AutoMLImageTrainingJob(display_name=display_name)
+
+    my_image_ds = aiplatform.Dataset(dataset_id)
```

Comment:
Since this is expecting dataset_id, we also need location or this could fail if the dataset is not in the default location ("us-central1")

Reply:
Added location param to be passed to init